docs/content/documentation/use.md

```diff
@@ -191,6 +191,10 @@ or 'statement XXX is not valid' so JDBC driver rolls back and retries
 * **`cleanupSavepoints (`*boolean*`)`** *Default `false`*\
 Determines if the SAVEPOINT created in autosave mode is released prior to the statement. This is done to avoid running out of shared buffers on the server in the case where 1000's of queries are performed.
 
+* **`convertBooleanToNumeric (`*boolean*`)`** *Default `false`*\
+Enable automatic conversion of PostgreSQL boolean values ('t'/'f') to numeric types (1/0) when using numeric getters (`getByte`, `getInt`, `getLong`, `getShort`) on ResultSet.
```

Comment:
Should probably update the comment here to include double, float, bigdecimal

Reply:
Updated.